jena-base/src/main/java/org/apache/jena/atlas/lib/cache/CacheSimple.java

```diff
@@ -37,120 +37,81 @@
 
 public class CacheSimple<K,V> implements Cache<K,V>
 {
-    private final V[] values ; 
-    private final K[] keys ;
-    private final int size ;
-    private int currentSize = 0 ;
-    private BiConsumer<K,V> dropHandler = null ;
+	private Map<K,V> internalCache;
+	
```

Comment:
LinkedHashMap implement an LRU policy and is what the old CacheLRU used.  We swapped that to CacheGuava and removed CacheLRU. Using LinkedHashMap again means we have two LRU caches.

CacheSimple has a specific policy of slot replacement should that be wanted.  We can leave it be and decide whether to delete it later.